source/index.ts

```diff
@@ -794,10 +794,33 @@ function typeErrorMessage(description: AssertionTypeDescription, value: unknown)
 	return `Expected value which is \`${description}\`, received value of type \`${is(value)}\`.`;
 }
 
-function typeErrorMessageMultipleValue(description: AssertionTypeDescription, values: unknown[]): string {
+function unique<T>(values: T[]): T[] {
 	// eslint-disable-next-line unicorn/prefer-spread
-	const valueTypes = Array.from(new Set(values.map(singleValue => `\`${is(singleValue)}\``))).join(', ');
-	return `Expected value which is \`${description}\`, received values of types ${valueTypes}.`;
+	return Array.from(new Set(values));
+}
+
+function joinWithWord(values: string[], word: 'and' | 'or'): string {
```

Comment:
Maybe use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/ListFormat

Reply:
Good idea, I switched to using `Intl.ListFormat` but ran into an issue where the type definitions did not exist for it.

I updated `tsconfig.json` to target ES2021.